backend/dataall/cdkproxy/stacks/policies/data_policy.py

```diff
@@ -102,7 +103,7 @@ def generate_data_access_policy(self) -> iam.Policy:
 
         return policy
 
-    def get_statements(self):
+    def get_statements(self, session):
```

Comment:
session is only used in the child class method, maybe replace it on this abstract level with args/kwargs?